android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -153,7 +158,52 @@ private void disableForegroundDispatch(NfcAdapter adapter) {
         adapter.disableForegroundDispatch(activity);
     }
 
+    static byte[] requireStatus(byte[] receivedApdu, byte[] expectedApdu) throws IOException {
```

Comment:
@petter-fidesmo but that means that it can't be expressed in a one liner like this:
byte[] response = requireStatus(transceive( .... ));
Which is a pretty nice feature. I do think that the boolean is more clear however. Let us go with that.